### PR TITLE
data connection to DS for getSymptoms endpoint

### PR DESCRIPTION
**Pull request by ChristinaLisa**

### CODE REVIEW

File: backend/models/symptoms.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const SymptomsSchema = new Schema({
+    symptomName: String
+});
```
Comment (rjeantet) [inline backend/models/symptoms.js:6]: There is no ID which might be why you currently have an error? Not sure,  but let's look into the bug together and solve it first so we are sure that no breaking code is being pushed to main.

**Comment by rjeantet** [comment]
It is crashing for me when testing.

Please:
- add the id param
- give me the URL where FE need to point to. (=> where in localhost/7000/**XXX** can you see the full list of symptoms?)

After that we try again

**Comment by rjeantet** [comment]
OK I added a backup call so I can approve now.
We still need the symptoms id's and I need to know whic URL I should point to in :7000

File: reactApp/src/data/getSymptoms.js [hunk 2]
```diff
@@ -1,23 +1,31 @@
 import axios from "axios";
 
+const dbUrl = "http://localhost:7000/getSymptoms";
```
Comment (rjeantet) [inline reactApp/src/data/getSymptoms.js:3]: This is not working. What is the proper url?

**Review by rjeantet — APPROVED** [review]

File: backend/models/symptoms.js (hunk 1 above)
Comment (ChristinaLisa) [reply backend/models/symptoms.js:6]: no that's urelated. You don't need to add ids to mongo schemas necesarily. Mongo does it on it's own. Also, Aljoscha saw the same issue sometimes.

**Pull request merged**